contracts/proxy/utils/Initializable.sol

```diff
@@ -41,11 +43,21 @@ abstract contract Initializable {
      */
     bool private _initializing;
 
+    /**
+     * @dev Modifier to protect (internal) functions that should only be executed as subcalls of an initializing
+     * function (protected with the {initializer} modifier). This modifier is designed to be used in upgradeable
+     * contracts for the constructor-replacing internal functions.
+     */
+    modifier onlyInitializing() {
```

Comment:
The goal is that we will update our Upgradeable Contracts to use `onlyInitializing` in place of `initializer`?

Reply:
Yes, I think its essential to support minimal proxy, and for that we need to use `onlyInitializing`